### PR TITLE
Allow to extract all items with a certain key tag.

### PR DESCRIPTION
**Pull request by Evelyn-M**
Changes to the `_query_builder()` (and the `extract()` function), to allow for an empty constraint clause, such that all items of a certain tag type can be extracted, irrespective of any further key-value constraint.

Fixes issue #20. 

Example: 
to parse all items from a file that have a non-empty "building" tag, one can now do:
`extract(OSM_FILE, 'multipolygons', ['building'])`

Tests amended accordingly.

### CODE REVIEW

**Comment by chahank** [comment]
Nice, thanks for the fix! Could you maybe add a minimal example to the documentation? Also, why did the figure `forest_che.png` change?

**Comment by Evelyn-M** [comment]
> 
The `forest_che.png` got there by accident; shouldn't have changed. will undo the commit. Can one undo just this addition in the commit?

**Comment by chahank** [comment]
I would say either do a cherry-picking or just make a new commit.

File: tests/test_extract.py [hunk 1]
```diff
@@ -35,6 +35,16 @@ def test_extract(self):
                          np.array(['MultiPolygon']))
         self.assertTrue(len(gdf_mp)==4202)
         
+        gdf_mp2 = extract(OSM_FILE, 'multipolygons',  ['building', 'name', ])
```
Comment (chahank) [inline tests/test_extract.py:38]: Is there a reason why all new tests have the value `name`? Is this needed?

**Comment by Evelyn-M** [comment]
> I would say either do a cherry-picking or just make a new commit.

worked, found out how to partially revert a commit.

File: tests/test_extract.py (hunk 1 above)
Comment (Evelyn-M) [reply tests/test_extract.py:38]: no, it works with any amount of keys. At least 1 is needed.

**Comment by Evelyn-M** [comment]
added an example on how to use the extract function without any value constraints now in the docs. Please merge if you think this is good enough @chahank

File: tests/test_extract.py (hunk 1 above)
Comment (chahank) [reply tests/test_extract.py:38]: Then a last question: is the empty element of the list intentional? I.e. the ', ' after the 'name' ?

Comment (Evelyn-M) [reply tests/test_extract.py:38]: no, you can remove it if you want :)

**Pull request merged**